content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md

```diff
@@ -59,51 +59,60 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
-          ./b2 -j <number of jobs>
+      5. To invoke Boost.Build to build the separately-compiled Boost libraries, run the following command. Replace `<number of parallel jobs>` with the number of jobs to run in parallel. Choose this value based on the number of CPU cores you want to use for building. This may take about 10 minutes, depending on your hardware specs.
 
-      f. Set the environment variable `BOOST_ROOT` to point to the new `boost_1_64_0` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
+              ./b2 -j <number of parallel jobs>
 
-          export BOOST_ROOT=/home/ubuntu/boost_1_64_0
+      6. Set the environment variable `BOOST_ROOT` to point to the new `boost_1_64_0` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
 
-      g. Source your updated `.profile` file. For example:
+              export BOOST_ROOT=/home/ubuntu/boost_1_64_0
 
-          source ~/.profile
+      7. Source your updated `.profile` file. For example:
 
-10. Get `rippled` source code.
+              source ~/.profile
 
+10. From a working directory, get the `rippled` source code.
+
+        cd ~
         git clone https://github.com/ripple/rippled.git
         cd rippled
         git checkout master
 
-11. If you previously built, or (more importantly) tried and failed to build `rippled`, you should delete the `build/` directory to start clean before moving on to the next step. Otherwise, you may get unexpected behavior, like a `rippled` executable that crashes due to a segmentation fault (segfault).
+11. If you previously built, or (more importantly) tried and failed to build `rippled`, you should delete the `my_build/` directory (or whatever you named it) to start clean before moving on to the next step. Otherwise, you may get unexpected behavior, like a `rippled` executable that crashes due to a segmentation fault (segfault).
+
+    If this is your first time building `rippled` 1.0.0 or higher, you won't have a `my_build/` directory and can move on to the next step.
+
+12. Use CMake to build a `rippled` binary executable from source code. The result will be a `rippled` binary executable in the `my_build` directory.
 
-    If this is your first time building `rippled,` you won't have a `build/` directory and can move on to the next step.
+      1. Generate the build system. Builds should be performed in a directory that is separate from the source tree root. In this example, we'll use a `my_build` directory that is a subdirectory of `rippled`.
 
-12. Build a `rippled` binary executable from source code. This may take about 20 minutes, depending on your hardware specs.
+              mkdir my_build
+              cd my_build
+              cmake -Dtarget=gcc.debug.unity ..
```

Comment:
1. Would it be better to recommend building a release version?
2. PR https://github.com/ripple/rippled/pull/2579 is going to change the way these parameters work. I don't know the details, but this is an early warning.

Reply:
1. Yeah, will add a step about checking out a release tag or something.
2. Noted.

Reply:
After digging deeper, I noticed the instructions already include the step of checking out the master branch, but I added a step to confirm the commit ID. (See 7df5717 for that.)